<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionquit"/>
    <addaction name="actionoff"/>
    <addaction name="actionswitch"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionzoom"/>
    <addaction name="actionlager"/>
    <addaction name="actiondelete_one"/>
    <addaction name="actioninvert"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>画图</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>矩形</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionempty_rect"/>
     <addaction name="actionfull_rect"/>
    </widget>
    <addaction name="actionDrawStraight"/>
    <addaction name="actionDrawCircle"/>
    <addaction name="actionDrawEllipse"/>
    <addaction name="actioncurve"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionfill"/>
    <addaction name="actionpaint"/>
    <addaction name="actioncrop"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawStraight"/>
   <addaction name="actionempty_rect"/>
   <addaction name="actionfull_rect"/>
   <addaction name="actionDrawCircle"/>
   <addaction name="actioncurve"/>
   <addaction name="actionDrawEllipse"/>
   <addaction name="separator"/>
   <addaction name="actionzoom"/>
   <addaction name="actionlager"/>
   <addaction name="separator"/>
   <addaction name="actioncrop"/>
   <addaction name="actionfill"/>
   <addaction name="actionpaint"/>
   <addaction name="separator"/>
   <addaction name="actiondelete_one"/>
   <addaction name="actiondelete"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionswitch"/>
   <addaction name="actioninvert"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionanother_save">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionDrawStraight">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/arrow_298px_1199168_easyicon.net.png</normaloff>:/png/arrow_298px_1199168_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="actionDrawCircle">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/polygom.png</normaloff>:/png/polygom.png</iconset>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
  </action>
  <action name="actionDrawEllipse">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/ellipse_stroked.png</normaloff>:/png/ellipse_stroked.png</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="actionzoom">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/zoom12.png</normaloff>:/png/zoom12.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionlager">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/zoom9.png</normaloff>:/png/zoom9.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionmax">
   <property name="text">
    <string>最大化</string>
   </property>
  </action>
  <action name="actionmin">
   <property name="text">
    <string>最小化</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionfill">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/paint_bucket.png</normaloff>:/png/paint_bucket.png</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
  </action>
  <action name="actionpaint">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/rotate2.png</normaloff>:/png/rotate2.png</iconset>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="actioncrop">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/crop.png</normaloff>:/png/crop.png</iconset>
   </property>
   <property name="text">
    <string>裁剪</string>
   </property>
  </action>
  <action name="actionbrush">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/paint.png</normaloff>:/png/paint.png</iconset>
   </property>
   <property name="text">
    <string>刷子</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/palette.png</normaloff>:/png/palette.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/trash37.png</normaloff>:/png/trash37.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="iconText">
    <string>清除</string>
   </property>
  </action>
  <action name="actiondelete_one">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/undo9.png</normaloff>:/png/undo9.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionempty_rect">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/design_280px_1199322_easyicon.net.png</normaloff>:/png/design_280px_1199322_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>空矩形</string>
   </property>
  </action>
  <action name="actionfull_rect">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/upright1.png</normaloff>:/png/upright1.png</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
  </action>
  <action name="actioncurve">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/draw_wave.png</normaloff>:/png/draw_wave.png</iconset>
   </property>
   <property name="text">
    <string>曲线</string>
   </property>
  </action>
  <action name="actionoff">
   <property name="text">
    <string>导入off</string>
   </property>
  </action>
  <action name="actionswitch">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/switch.png</normaloff>:/png/switch.png</iconset>
   </property>
   <property name="text">
    <string>切换</string>
   </property>
  </action>
  <action name="actioninvert">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/png/invert.png</normaloff>:/png/invert.png</iconset>
   </property>
   <property name="text">
    <string>反选</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
